Description: Adjust installation paths in CMakeLists.txt
 Adjust the installation paths in CMakeLists.txt to match the
 paths used in Debian:
 - The udev-rules should be installed into /lib/udev/rules.d.
Author: Jan Luca Naumann <j.naumann@fu-berlin.de>
Forwarded: not-needed
Last-Update: 2018-05-13
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -129,8 +129,6 @@
         ELSE()
             set(CMAKE_INSTALL_UDEVRULESDIR "${CMAKE_INSTALL_UDEVRULESDIR}/rules.d")
         ENDIF()
-        string(REGEX REPLACE "^/" "" CMAKE_INSTALL_UDEVRULESDIR "${CMAKE_INSTALL_UDEVRULESDIR}")
-        string(REGEX REPLACE "^usr/" "" CMAKE_INSTALL_UDEVRULESDIR "${CMAKE_INSTALL_UDEVRULESDIR}") # usual prefix is usr/local
         message(STATUS "Setting udev rules dir to ${CMAKE_INSTALL_UDEVRULESDIR}")
     ENDIF()
 
